data/battle_anim_scripts.s

```diff
@@ -17159,6 +17159,33 @@ Move_RAGING_BULL::
 	restorebg
 	waitbgfadein
 	end
+	
+@ Credits to Z-nogyroP. Simple anim that combines Force Palm + Fake Out
```

Comment:
I added that because the other anims I contributed were credited, but I'm happy to remove it

Reply:
Looks like Alex originally included the credit because he was the one PRing it rather than you. Upon a closer look, it looks like there's credits for at least two other people in the animation file too, so there's enough precedent to leave your name there :) (fingers crossed this doesn't evolve into a self-crediting epidemic)